activities/live-streaming/manage-live-stream.md

```diff
@@ -0,0 +1,28 @@
+---
+type: guide
+explains: how to run a live stream
+---
+
+# Managing a live stream
+
+When running a [live stream](index.md) of 'Let's talk about  public code' there are some steps to remember:
+
+* Test everything before going live.
+* Prepare tabs with easy access to moderation tools of the different platforms to be able to intervene in case of offensive commenting. Preferably, this is done by a staff member not in the actual interview.
+* Meet with the guest in [StreamYard](../tool-management/streamyard.md) 30 minutes before going live.
+* Keep an eye on attendees' questions and behavior: when are they connecting, which is the most successful platform and why, best timing, etc.
+* When it's time to start the stream, press Go live and quickly start the intro video.
+* Start by only having the interviewers on screen.
+* Remind the audience what the Foundation for Public Code does and how to contact us in an easy way.
+* Present the guest and bring them on screen.
+* Run the interview.
+* Bring up interesting questions from the audience on the screen and read them out loud to the interviewee.
+* Remember that there will be a podcast version afterwards, so make sure to explain things so that someone only listening to the show will understand (in case of shared screens or funny things happening on camera).
+* The last question we always ask our guest is "Please name someone you think we should interview in our upcoming episodes, and explain why?"
+* Remind the audience at the end of ways of keeping in touch with us.
+* When the show is over start the outro video and quickly press Stop recording.
```

Comment:
This on the other hand is a list of steps that I'm guessing should be taken in order and thus might be better served being ordered.

Reply:
cc @Ainali @AlbaRoza